src/elements/content-explorer/MetadataViewContainer.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import { MetadataView, type MetadataViewProps } from '@box/metadata-view';
+
+import type { SortDescriptor } from 'react-aria-components';
+import type { Collection } from '../../common/types/core';
+
+export interface MetadataViewContainerProps extends Omit<MetadataViewProps, 'items'> {
+    currentCollection: Collection;
+    onSortChange?: (sortBy: string, sortDirection: string) => void;
+}
+
+const MetadataViewContainer = ({
+    actionBarProps,
+    columns,
+    currentCollection,
+    onSortChange,
+    tableProps,
+    ...rest
+}: MetadataViewContainerProps) => {
+    const { items = [], metadataTemplate } = currentCollection;
+
+    const filterGroups = React.useMemo(
+        () => [
+            {
+                togglable: true,
```

Comment:
`toggleable` vs `togglable`

If just fixing this causes you to have to fight e2e tests 5 times in a row, I wouldn't mind leaving this alone.